extensions/wikia/Wall/index/CommentsIndexHooks.class.php

```diff
@@ -26,13 +26,17 @@ static public function onArticleDoDeleteArticleBeforeLogEntry( WikiPage $page, $
 		if ( $title->isTalkPage() && WallHelper::isWallNamespace( $title->getNamespace() ) ) {
 			wfDebug(__METHOD__ . "\n\n");
 
-			// we have either Wall message or Forum post - mark comments_index entry
-			$wallMessage = WallMessage::newFromTitle( $title );
-			$wallMessage->setInCommentsIndex( WPP_WALL_ADMINDELETE, 1 ); // set deleted = 1
+			try {
+				// we have either Wall message or Forum post - mark comments_index entry
+				$wallMessage = WallMessage::newFromTitle( $title );
+				$wallMessage->setInCommentsIndex( WPP_WALL_ADMINDELETE, 1 ); // set deleted = 1
 
-			// invalidate message and thread cache
-			$wallMessage->getThread()->invalidateCache();
-			$wallMessage->invalidateCache();
+				// invalidate message and thread cache
+				$wallMessage->getThread()->invalidateCache();
+				$wallMessage->invalidateCache();
+			} catch ( CommentsIndexEntryNotFoundException $notFoundException ) {
+				// bummer
```

Comment:
Do we log these exceptions?

Reply:
we already do in other places, but we can log these occurrences too